JSON.md

```diff
@@ -29,11 +29,24 @@ As an example, here is a local policy file module with 3 common build steps:
   "description": "Local policy file added using cfbs command line",
   "tags": ["local"],
   "added_by": "cfbs add",
-  "steps": [
-    "copy ./policy.cf services/cfbs/policy.cf",
-    "policy_files ./services/cfbs/policy.cf",
-    "bundles my_bundle"
-  ]
+  "build": [
+    {
+      "name": "cusom policy",
+      "description": "Custom policy basic setup",
+      "tags": [],
+      "repo": "https://github.com/cfengine/modules",
+      "by": "https://github.com/cfengine",
+      "version": "0.0.1",
+      "commit": "ca637d4e6148432a90b7db598a4137956c0e0282",
+      "added_by": "cfbs add",
+      "steps": [
+        "copy ./policy.cf services/cfbs/policy.cf",
+        "policy_files ./services/cfbs/policy.cf",
+        "bundles my_bundle"
+      ]
+    }
+  ],
+  "git": false
```

Comment:
This example is not a full `cfbs.json` file, just one module. We can add a clarifying sentence about this, but this change doesn't make much sense. (`tags` and `added_by` should be at the module level, not the project / `cfbs.json` top level, the name of a custom policy file should be something like it was `./policy.cf`, and it should not have version, repo, commit SHA, etc.).

Reply:
https://github.com/cfengine/cfbs/pull/176